frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import { ReactComponent as ResultIcon } from '../icons/ResultIcon.svg'
+import { ReactComponent as StatementIcon } from '../icons/StatementsIcon.svg'
+
+export default function Navbar() {
+  return (
+    <>
+      <NavbarStyled>
+        <LinkStyled className="nav" activeClassName="selected" to="/statements">
+          <StatementIconStyled />
+        </LinkStyled>
+        <LinkStyled className="nav" activeClassName="selected" to="/result">
```

Comment:
brauchst du die klasse nav? Unten in den Styles finde ich nur die selected-Klasse.

Reply:
danke, habe es aufgeräumt